src/components/v5/common/ActionsContent/partials/TeamsSelect/TeamsSelect.tsx

```diff
@@ -1,22 +1,61 @@
-import React, { FC } from 'react';
+import React, { FC, useState } from 'react';
+import { useIntl } from 'react-intl';
 
+import { useFormContext } from 'react-hook-form';
 import { useTeamsSelect } from './hooks';
 import SearchSelect from '~v5/shared/SearchSelect/SearchSelect';
+import useToggle from '~hooks/useToggle';
+import TeamBadge from '~v5/common/Pills/TeamBadge';
+import styles from '../../ActionsContent.module.css';
 import { SelectProps } from '../../types';
 
 const displayName = 'v5.common.ActionsContent.partials.TeamsSelect';
 
-const TeamsSelect: FC<SelectProps> = ({ isOpen, onToggle, onSelect }) => {
+const TeamsSelect: FC<SelectProps> = ({ name }) => {
+  const { register, setValue } = useFormContext();
   const teamsOptions = useTeamsSelect();
+  const { formatMessage } = useIntl();
+  const [selectedTeam, setSelectedTeam] = useState<string | null>(null);
+  const [isTeamSelectVisible, { toggle: toggleTeamSelect }] = useToggle();
 
-  return isOpen ? (
-    <SearchSelect
-      items={[teamsOptions]}
-      isOpen={isOpen}
-      onToggle={onToggle}
-      onSelect={onSelect}
-    />
-  ) : null;
+  return (
+    <>
```

Comment:
TeamsSelect component and UserSelect look the similar, please try move components to separate files which can be extracted